static/app/views/performance/transactionSummary/transactionSpans/spanDetails/utils.tsx

```diff
@@ -47,3 +47,4 @@ export function spanDetailsRouteWithQuery({
 
 export const MIN = 'min';
 export const MAX = 'max';
+export const ZOOM_KEYS = [MIN, MAX];
```

Comment:
A bit of a nitpick but imo I think this would prob be nicer as an enum, that way you can just access `ZOOM_KEYS.MIN`, `ZOOM_KEYS.MAX` instead of exporting both of them separately and an array here. Definitely not necessary though, since this works fine 🙂

Reply:
done!